ods_ci/tasks/Resources/Provisioning/GPU/AMD/amd_operator.sh

```diff
@@ -146,34 +147,150 @@ function wait_until_driver_image_is_built() {
   fi
 }
 
+function label_amd_gpu_nodes() {
+  echo "labeling"
+  echo "Checking for AMD GPU nodes and adding required labels..."
+  
+  # Find AMD GPU nodes by instance type
+  AMD_GPU_NODES=$(oc get nodes -o json | jq -r '.items[] | select(.metadata.labels."node.kubernetes.io/instance-type" | test("g4ad|g5ad")) | .metadata.name')
+  
+  if [[ -z "$AMD_GPU_NODES" ]]; then
+    echo "No AMD GPU nodes found (looking for g4ad.* or g5ad.* instance types)"
+    echo "Available nodes:"
+    oc get nodes -o json | jq -r '.items[] | "\(.metadata.name): \(.metadata.labels."node.kubernetes.io/instance-type")"'
+    return 0
+  fi
+  
+  # Label each AMD GPU node
+  for node in $AMD_GPU_NODES; do
+    echo "Processing AMD GPU node: $node"
+    
+    # Check if node already has the required label
+    if oc get node $node --show-labels | grep -q "feature.node.kubernetes.io/pci-1002.present=true"; then
+      echo "  Node $node already has AMD PCI label"
+    else
+      echo "  Adding AMD PCI label to node $node"
+      oc label node $node feature.node.kubernetes.io/pci-1002.present=true
+      if [[ $? -eq 0 ]]; then
+        echo "  ✅ Successfully labeled node $node"
+      else
+        echo "  ❌ Failed to label node $node"
+      fi
+    fi
+  done
+  
+  # Display labeled nodes
+  echo "AMD GPU nodes with PCI labels:"
+  oc get nodes -l feature.node.kubernetes.io/pci-1002.present=true -o custom-columns="NAME:.metadata.name,INSTANCE_TYPE:.metadata.labels.node\.kubernetes\.io/instance-type"
+}
+
 function create_acceleratorprofile() {
-  echo "Creating AMD Accelerator Profile"
-  rhoai_ns=$(oc get namespace redhat-ods-applications --ignore-not-found  -oname)
-  if [ -z $rhoai_ns ];
-    then
-      echo "redhat-ods-applications namespace not found. Is RHOAI Installed? NVIDIA Accelerator Profile creation SKIPPED."
-      return 0
+  echo "Creating AMD Hardware/Accelerator Profile for OpenDataHub/RHOAI"
```

Comment:
could you please verify if we need the same fix for NVIDIA GPUs?

https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tasks/Resources/Provisioning/GPU/NVIDIA/gpu_deploy.sh#L53